README.rst

```diff
@@ -1,81 +1,98 @@
-This package contains the Zephyr RTOS meta tool, 'west'.
+This is the Zephyr RTOS meta tool, ``west``.
 
-WARNING
--------
+Installation
+------------
 
-DO NOT INSTALL WITH "python3 setup.py install".
+Install West's bootstrapper with pip::
 
-Please use pip to install in development mode as documented below.
+  pip3 install west
```

Comment:
One of the things I feel are extremely useful and sorely miss on Windows cmd line is git bash autocomplete. I wonder if we could reuse their script somehow and actually install it during bootstrapping.

Reply:
Taking a serious look at autocomplete on all the platforms is on my wishlist as well -- should we start backlogging this in the issues for this repo?

Reply:
Yes actually. I will create one right now.